### ISSUE TITLE
organization_id needs to be returned on a GET request to /orgs.

### ISSUE THREAD

**Issue by ryanmdoyle** [code4sac/neighborhood-connect-api#24]
I'm working on the front end right now and we are going to either need the organization_id (I noticed that's part of what we get with the User) or a unique ID.

Basically, our forms and filters have access to the name of the organizations, but it would be better to use the org id to identify them in case there are similar/changing org names later on.

**Comment by ryanmdoyle** [comment]
To clarify, we have the following response currently:
```
{
            "name": "Avondale Glen Elder Neighborhood Association",
            "district": "6",
            "logo_url": null
        },
```
The district might be uniquely identifiable?  If it is then we can use that, but if there are multiple associations/neighborhoods within a district, we need some way to identify this one specifically (aside from relying on the "name".

**Comment by ryanmdoyle** [comment]
Fixed with #29
